components/text-input-field/TextInputField.style.js

```diff
@@ -1,16 +1,30 @@
 import styled from 'styled-components';
 import { P } from '../../style/typography';
+import { min } from '../../lib/responsive';
 import { colors } from '../../style/colors';
 
 const TextLabel = styled(P)`
   color: ${colors.BROWN};
   display: block;
   margin: 0;
+  font-size: 0.8em;
+`;
+
+const ErrorText = styled(P)`
+  position: absolute;
+  color: red;
+  font-size: 0.7em;
+  margin: 0.5em 0 0 0;
+  text-shadow: 0.5px 0 #ff0000;
+  letter-spacing: 0.05em;
 `;
 
 const RedStar = styled.img`
   width: 2%;
```

Comment:
I know it wasn't in this PR but the red star looks slightly too large on the desktop sizes